doc/source/extended_documentation/calibration/beta_recalibration/beta_recalibration.rst

```diff
@@ -7,8 +7,19 @@ calibrated (for example, by reliability calibration or by the Rainforests calibr
 that, when blending probabilistic forecasts, even if each input is perfectly calibrated, the output is in general not perfectly calibrated. 
 The authors also show that applying a recalibration to the blended output improves its reliability, sharpness, and score on proper scoring metrics. 
 IMPROVER implements the recalibration method studied in the 
-article, namely a transformation given by the cumulative distribution function of the beta distribution. 
+article, namely a transformation given by the cumulative distribution function of the beta distribution. This is a natural choice of transformation 
+function for probabilities because it is a monotone-increasing function that maps the interval `[0, 1]` onto itself.
 The implementation here allows the `alpha` and `beta` parameters of the beta distribution to vary by forecast period.
 
+It is recommended that the `alpha` and `beta` parameters be chosen to optimise the desired metric (for example, the continuous rank
+probability score) over some training period. Specifically, this could be done as follows:
+
+1. Obtain blended forecast output, along with ground truth data from observations or analyses, for the training period.
+2. Implement the loss function. The loss function should have parameters `alpha` and `beta` and return the loss over the training period when the probabilistic forecast is transformed by the CDF of the beta distribution function. A suggested loss function is the CRPS calculated from the thresholded probability forecast.  In this case, the loss function should transform the input blended probabilities by the CDF of the beta distribution (which is available in `scipy.stats`), then calculate the CRPS of the transformed probability forecast against the ground truth.
```

Comment:
I wonder if `arguments` is a better fit here? `parameter` is suitable, but `arguments` suggests that values are passed through the loss function to return a value.
```suggestion
2. Implement the loss function. The loss function should have arguments `alpha` and `beta` and return the loss over the training period when the probabilistic forecast is transformed by the CDF of the beta distribution function. A suggested loss function is the CRPS calculated from the thresholded probability forecast. In this case, the loss function should transform the input blended probabilities by the CDF of the beta distribution (which is available in `scipy.stats`), then calculate the CRPS of the transformed probability forecast against the ground truth.
```

Reply:
Changed as suggested